velox/exec/fuzzer/WriterFuzzer.cpp

```diff
@@ -259,10 +319,27 @@ void WriterFuzzer::go() {
     std::vector<TypePtr> types;
     std::vector<std::string> bucketColumns;
     std::vector<std::string> partitionKeys;
+    std::vector<std::shared_ptr<const HiveSortingColumn>> sortBy;
 
     // Regular table columns
     generateColumns(5, "c", kRegularColumnTypes_, 2, names, types);
 
+    // TODO: sort columns can overlap as bucket columns
+    // 50% of times test ordered write.
+    const auto sortColumnOffset = names.size();
```

Comment:
Shall we only set this after we decide the bucket and partition columns? thanks!